### PR TITLE
feat[STK-3]: add Stackly as a safe app

### PR DESCRIPTION
**Pull request by berteotti**
PR adds logic and manifest.json so Stackly can be used as a Safe App. 

Follow this tutorial to create a custom app using the preview link https://help.safe.global/en/articles/40859-add-a-custom-safe-app .

### CODE REVIEW

**Comment by Diogomartf** [comment]
why do we need auto connect?

**Comment by berteotti** [comment]
> why do we need auto connect?

Auto connect is used to override wagmi auto connection logic. In this case we try to connect Safe first, so that you always auto connect to Safe inside Safe App.

**Comment by Diogomartf** [comment]
> > why do we need auto connect?
> 
> Auto connect is used to override wagmi auto connection logic. In this case we try to connect Safe first, so that you always auto connect to Safe inside Safe App.

Can we make it safe only, and don't do it on our main app?

File: packages/app/hooks/autoConnect.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      AUTOCONNECTED_CONNECTOR_IDS.every((connector) => {
+        const connectorInstance = connectors.find(
+          (c) => c.id === connector && c.ready
+        );
+
+        if (connectorInstance) {
+          connect({ connector: connectorInstance });
+          return false;
+        }
+        return true;
```
[the remark this answers is not in the document]
Comment (berteotti) [reply packages/app/hooks/autoConnect.ts:29]: We could use other type of loops, but using `every` works too. We just need to return `true` when we want to iterate further and `false` when we need to stop. `break` or something similar, should work with `forEach` but it doesn't. We can work with for ... in if you find it better.

**Pull request merged**